subprojects/packagefiles/libcbor/meson.build

```diff
@@ -0,0 +1,161 @@
+project(
+  'libcbor',
+  'c',
+  version: '0.13.0',
+  meson_version: '>=0.59.0',
+  default_options: ['warning_level=3'],
+  license: 'MIT',
+)
+
+cc = meson.get_compiler('c')
+
+tests_opt = get_option('tests').disable_auto_if(meson.is_subproject())
```

Comment:
Why disable if it's a subproject?

Reply:
If the tests are left enabled in a subproject they'll all be built when running `ninja install`, which can be time consuming or at worst lead to build failures on stranger build configurations.